lib/ansible/parsing/mod_args.py

```diff
@@ -107,12 +111,9 @@ def __init__(self, task_ds=None, collection_list=None):
             raise AnsibleAssertionError("the type of 'task_ds' should be a dict, but is a %s" % type(task_ds))
         self._task_ds = task_ds
         self._collection_list = collection_list
-        # delayed local imports to prevent circular import
-        from ansible.playbook.task import Task
-        from ansible.playbook.handler import Handler
         # store the valid Task/Handler attrs for quick access
-        self._task_attrs = set(Task.fattributes)
-        self._task_attrs.update(set(Handler.fattributes))
+        self._task_attrs = set(playbook_task.Task.fattributes)
+        self._task_attrs.update(set(playbook_handler.Handler.fattributes))
```

Comment:
Why not perform the import in `__init__` instead? That requires no support code and should be better understood by static analysis tools.

Reply:
This specific file wasn't my use case, I just randomly picked a single file to update. I had this written for a different use case, and saw we have FIXMEs and TODOs, at least in a couple of PRs to remove imports within functions, so figured I'd just push this up.

Reply:
The use of non-global imports is sub-optimal, and a good sign the code should be refactored to avoid circular imports. However, if we're not fixing the underlying circular reference, I think using normal non-global imports is preferable to lazy imports.